buildSrc/src/main/java/Configurations.kt

```diff
@@ -1,3 +1,14 @@
+object Flavors {
+    const val PRODUCTION = "production"
+    const val STAGING = "staging"
+    const val DIMENSION = "stage"
```

Comment:
What is the Dimension flavor? should its value be `"dimension"`?

Reply:
@blyscuit It's a required [build-variant](https://developer.android.com/studio/build/build-variants#product-flavors) config. I changed it to `DIMENSION_VERSION = "version"` https://github.com/luongvo/kmm-survey/pull/42/commits/4e2294990cdd0be98308ad223e1ac68491145a44.